compiler/x/codegen/BinaryEvaluator.cpp

```diff
@@ -1252,8 +1252,7 @@ TR::Register *OMR::X86::TreeEvaluator::caddEvaluator(TR::Node *node, TR::CodeGen
       }
 
    TR::Register * testRegister = secondChild->getRegister();
-   if (secondChild->getOpCodeValue() == TR::cconst &&
-       testRegister == NULL         &&
+   if (testRegister == NULL         &&
```

Comment:
Is this logic what you intended here ? I ask because the opcode value not being ```cconst``` suggests that this if condition ought not to be entered.

Reply:
I believe you are reviewing the outdated version of this. I have deleted this whole if statement.